pilot/pkg/kube/inject/webhook.go

```diff
@@ -26,8 +26,11 @@ import (
 	"sync"
 	"time"
 
+	ocprom "contrib.go.opencensus.io/exporter/prometheus"
```

Comment:
is this what we use in other places in istio?

Reply:
Yes. Pilot monitoring uses it like that https://github.com/istio/istio/blob/master/pilot/pkg/bootstrap/monitoring.go#L41